chemicaltoolbox/rdock/rbdock.xml

```diff
@@ -0,0 +1,286 @@
+<tool id="rbdock" name="rDock docking" version="0.1">
+    <description>- perform protein-ligand docking with rDock</description>
+    <requirements>
+        <requirement type="package" version="2013.1-0">rdock</requirement>
+    </requirements>
+    <command><![CDATA[
+ln -s '$active_site' receptor.as &&
+ln -s '$receptor' receptor.mol2 &&
+ln -s $receptor_prm receptor.prm &&
+#if $name == 'Y':
+  sdmodify -f_REC '$ligands' > ligands.sdf &&
+#else
+  ln -s '$ligands' ligands.sdf &&
+#end if
+rbdock -i ligands.sdf -r receptor.prm -p dock.prm -n $num -o output &&
+sdsort -n -s -fSCORE output.sd |
+#if len(str($score)) > 0:
```

Comment:
`#if $score and $score > 0` should work.

Reply:
Changed syntax: https://github.com/tdudgeon/galaxytools/commit/3fe85518879a9a987112f46eb10d2a028575d973